interop-tests/src/main/scala/akka/grpc/interop/AkkaGrpcClientScala.scala

```diff
@@ -22,6 +25,16 @@ final case class AkkaGrpcClientScala(clientTesterFactory: Settings => ActorSyste
     val client = new TestServiceClient(clientTesterFactory(settings)(sys))
     client.setUp()
 
+    // alternatively `.withSslContext` directly from a SSLContextFactory.createSSLContextFromPem if not
+    // rotating certs
+    GrpcClientSettings
+      .fromConfig(sys.settings.config)
+      .withSslContextProvider(SSLContextFactory.refreshingSSLContextProvider(5.minutes)(() =>
+        SSLContextFactory.createSSLContextFromPem(
+          certificatePath = Paths.get("/some/path/server.crt"),
+          privateKeyPath = Paths.get("/some/path/server.key"),
+          trustedCaCertificatePaths = Seq(Paths.get("/some/path/serverCA.crt")))))
+
```

Comment:
Temporary API test for now, doesn't actually use it. I need to add some proper test for that, after verifying it actually works at all.